samples/peripheral/802154_phy_test/Kconfig

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "PHY Test Tool configuration"
+
+choice
+	prompt "Device Mode"
+	default BOTH
+	help
+	  Select the device mode
+
+config DUT_MODE
+	bool "Configure device as a DUT"
+
+config CMD_MODE
+	bool "Configure device as a CMD"
+
+config BOTH
+	bool "Supports both CMD and DUT modes"
+
+endchoice
+
+config POWER
+	int "Transmit Power (dBm)"
+	default -20
```

Comment:
Why default TX power is so low?
I think it would be better to add help string here to describe what values are correct. I believe it's not OK to set any int here.